content/blogposts/2022/release-3.41.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Sourcegraph 3.41 release'
+publishDate: 2022-06-21T10:00-07:00
+description: 'Sourcegraph 3.41 introduces running batch changes server-side, file mounting for the batch change container, greater pattern tracking in Code Insights, and admin settings to restrict extensions to those authored by Sourcegraph.'
+tags: [blog, release]
+slug: 'release/3.41'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/release-post/3.41/sourcegraph-3-41-release.png
+changelogItems: 
+  - description: 'Added a new templating variable, batch_change_link, to give more control over where the "Created by Sourcegraph batch change ..." message appears in the published changeset description. This message would occasionally break CI workflows that parsed the bottom of a message; it can now be moved accordingly.'
```

Comment:
This actually shows up as "Created by Sourcegraph batch change _X_" where X is the name of the batch change. We can drop the ellipses since that's not a literal part of the message

Reply:
```suggestion
  - description: 'Added a new templating variable, batch_change_link, to give more control over where the "Created by Sourcegraph batch change" message appears in the published changeset description. This message would occasionally break CI workflows that parsed the bottom of a message; it can now be moved accordingly.'
```